bundle/Resources/config/field_types.yaml

```diff
@@ -1,9 +0,0 @@
-services:
-    ngsite.field_type.ezrichtext.converter.direct_download:
-        class: Netgen\Bundle\SiteBundle\RichText\Converter\LinkDirectDownload
```

Comment:
Do we need to delete this class?

Reply:
deleted in a282543a6e9d7949afd578ae6dbc60ee2d06e1f3, I thought i already deleted it, sry.